dev/deps/spark-deps-hadoop-2.4

```diff
@@ -98,7 +98,7 @@ jersey-client-2.22.2.jar
 jersey-common-2.22.2.jar
 jersey-container-servlet-2.22.2.jar
 jersey-container-servlet-core-2.22.2.jar
-jersey-guava-2.22.2.jar
+jersey-guava-2.22.2.jarshaded-proto
```

Comment:
Any particular reason why this changed? Would be curious to trace how this dependency changed.